### ISSUE TITLE
Time zone for "GTB Standard Time" wrong?

### ISSUE THREAD

**Issue by dg76** [nfergu/Java-Time-Zone-List#4]
Hi!

I think the correct time zone translation for "GTB Standard Time" should be "Europe/Athens" according to the Microsoft documentation here:

https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/default-time-zones

Do you agree? 🤔

**Comment by flaviorighi** [comment]
I agree with @dg76 and Istanbul should be removed too from the group "(GMT +02:00) Athens, Bucharest", because is GMT +03:00 there.

I've added the row:
`TIMEMAPPINGS.add(new TimeZoneMapping("Turkey Time", "Europe/Istanbul", "(GMT +03:00) Istanbul"));`

And edited this one:
`TIMEMAPPINGS.add(new TimeZoneMapping("GTB Standard Time", "Europe/Athens", "(GMT +02:00) Athens, Bucharest"));`
